### PR TITLE
clarify cwnd growth in convex region

### LINKED ISSUE #69: Overridden by linear growth by AIMD
[Yoshi said](https://mailarchive.ietf.org/arch/msg/tcpm/W-t1q8q6Q0Ofjcp0iXOgmY18mr0):
>> 10: Section 4.5: " The convex profile ensure that the window increases very slowly at the beginning.."
>
>   I am wondering how much this part is accurate. Because of Principal 2,  even though cwnd is increased 
   through convex profile, I think it will be overridden by linear growth by AIMD.

### PR DESCRIPTION
**Pull request by goelvidhi**
Closes #69

### CODE REVIEW

**Review by lisongxu — COMMENTED** [review]
"Unless it is not overridden by AIMD window increase, CUBIC is very careful in this region" should be "Unless it is overridden by AIMD window increase, CUBIC is very careful in this region"? Thank you

File: draft-ietf-tcpm-rfc8312bis.md [hunk 1]
```diff
@@ -613,11 +613,13 @@ available bandwidth after some flow departures. Since the Internet is
...
 without causing a major change in available bandwidth.
 
-In this region, CUBIC is very careful. The convex profile ensures that
-the window increases very slowly at the beginning and gradually
-increases its increase rate. We also call this region the "maximum
-probing phase", since CUBIC is searching for a new *W<sub>max</sub>*.
-In this region, *cwnd* MUST be incremented by
+Unless it is not overridden by AIMD window increase, CUBIC is very
+careful in this region. The convex profile aims that
+the window increases very slowly at the beginning when the *cwnd* is
```
Comment (larseggert) [inline draft-ietf-tcpm-rfc8312bis.md:618]:
```suggestion
careful in this region. The convex profile aims to
increase the window very slowly at the beginning when *cwnd* is
```

File: draft-ietf-tcpm-rfc8312bis.md [hunk 2]
```diff
@@ -613,11 +613,13 @@ available bandwidth after some flow departures. Since the Internet is
...
+around *W<sub>max</sub>* and then gradually increases its increase rate.
```
Comment (larseggert) [inline draft-ietf-tcpm-rfc8312bis.md:619]:
```suggestion
around *W<sub>max</sub>* and then gradually increases its rate of increase.
```

File: draft-ietf-tcpm-rfc8312bis.md [hunk 3]
```diff
@@ -613,11 +613,13 @@ available bandwidth after some flow departures. Since the Internet is
 highly asynchronous, some amount of perturbation is always possible
 without causing a major change in available bandwidth.
 
-In this region, CUBIC is very careful. The convex profile ensures that
-the window increases very slowly at the beginning and gradually
-increases its increase rate. We also call this region the "maximum
-probing phase", since CUBIC is searching for a new *W<sub>max</sub>*.
-In this region, *cwnd* MUST be incremented by
+Unless it is not overridden by AIMD window increase, CUBIC is very
```
Comment (larseggert) [inline draft-ietf-tcpm-rfc8312bis.md:616]:
```suggestion
Unless it is not overridden by the AIMD window increase, CUBIC is very
```

**Review by larseggert — CHANGES_REQUESTED** [review]

**Comment by goelvidhi** [comment]
> "Unless it is not overridden by AIMD window increase, CUBIC is very careful in this region" should be "Unless it is overridden by AIMD window increase, CUBIC is very careful in this region"? Thank you

done

**Review by lisongxu — APPROVED** [review]
Thanks!

**Review by nsdyoshi — APPROVED** [review]

**Review by larseggert — APPROVED** [review]

**Pull request merged**